azkaban-db/src/main/sql/migrations/patch-1.sql

```diff
@@ -0,0 +1,3 @@
+-- DB Migration from release 3.83.0
+-- Increasing the side of json column in project_flows table.
+ALTER TABLE project_flows MODIFY COLUMN json MEDIUMBLOB;
```

Comment:
Should we also change the CREATE TABLE statement as part of this?

Reply:
We should not. That is how patch should work.

Reply:
Not sure how it has been done historically in Azkaban. 
What happens when a new Azkaban cluster is being initialized ? Is the schema created by always running the original schema creation along with all the patches. Does not seem like a good way of doing this. 
(Not suggesting it needs to be changed at this point - only confirming the current state of things)

Reply:
Totally agree with you. There is no proper process right now. But it will be good to follow some standards for database patches. We can discuss about it and implement it.

Reply:
I believe that in cases of schema modifications there is currently a convention of adding them to a file with name like "upgrade.3.xx.0.to.3.yy.0" and since there no mechanism in place for applying changes in those files to newly created DBs automatically we also add the schema change to the create.<table-name>.sql files.